<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpecGAN: 光谱图像转换工具</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* ---- 全局样式 ---- */
#MainWindow {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(27, 38, 59, 255), stop:1 rgba(13, 19, 31, 255));
}

QStatusBar {
    background-color: rgba(27, 38, 59, 0.8);
    color: #9FB6D9;
    font-size: 13px;
    font-weight: bold;
}

QToolTip {
    background-color: #1B263B;
    color: #EAEAEA;
    border: 1px solid #40E0D0;
    border-radius: 4px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QFrame" name="topFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="selectButton">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>从本地磁盘选择一张图片作为输入</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: #EAEAEA;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 129, 255, 150), stop:1 rgba(0, 255, 255, 180));
    border: 1px solid #00FFFF;
    border-radius: 8px;
}
QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 129, 255, 200), stop:1 rgba(0, 255, 255, 220));
    border: 1px solid #99FFFF;
}
QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 100, 200, 255), stop:1 rgba(0, 200, 200, 255));
}
QPushButton:disabled {
    background-color: #555;
    color: #888;
    border: 1px solid #666;
}</string>
         </property>
         <property name="text">
          <string>① 选择图像</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="runButton">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>启动NPU，开始执行图像转换</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 255, 127, 180), stop:1 rgba(0, 200, 200, 200));
    border: 1px solid #00FF7F;
    border-radius: 8px;
}
QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 255, 127, 220), stop:1 rgba(0, 200, 200, 240));
    border: 1px solid #80FFBF;
}
QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 200, 100, 255), stop:1 rgba(0, 150, 150, 255));
}
QPushButton:disabled {
    background-color: #555;
    color: #888;
    border: 1px solid #666;
}</string>
         </property>
         <property name="text">
          <string>② 开始转换</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>将右侧生成的图片保存到本地</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: #FFFFFF;
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 165, 0, 180), stop:1 rgba(255, 69, 0, 200));
    border: 1px solid #FFA500;
    border-radius: 8px;
}
QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 165, 0, 220), stop:1 rgba(255, 69, 0, 240));
    border: 1px solid #FFC04D;
}
QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(204, 132, 0, 255), stop:1 rgba(204, 55, 0, 255));
}
QPushButton:disabled {
    background-color: #555;
    color: #888;
    border: 1px solid #666;
}</string>
         </property>
         <property name="text">
          <string>③ 保存图像</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="imageFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="inputImageLabel">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: rgba(0, 0, 0, 0.4);
    border: 2px dashed #4A5A73;
    border-radius: 12px;
    color: rgba(127, 140, 141, 0.7);
}
QLabel:hover {
    border: 2px dashed #40E0D0;
}</string>
         </property>
         <property name="text">
          <string>Input</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="outputImageLabel">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>24</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background-color: rgba(0, 0, 0, 0.4);
    border: 2px dashed #4A5A73;
    border-radius: 12px;
    color: rgba(127, 140, 141, 0.7);
}</string>
         </property>
         <property name="text">
          <string>Output</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
    background-color: #1B263B;
    color: #EAEAEA;
    font-weight: bold;
}

QMenuBar::item:selected {
    background-color: #3498db;
}</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
